homeassistant/components/alarmdecoder.py

```diff
@@ -116,15 +112,33 @@ def stop_alarmdecoder(event):
     @callback
```

Comment:
This callback is not run in the event loop, since the callback is called from another thread. So remove this annotation.

Reply:
How do I know what thread the hass.bus.listen_once callback comes from?  Is it not the main event loop?